evalml/tests/pipeline_tests/test_time_series_baseline_pipeline.py

```diff
@@ -73,3 +74,41 @@ def test_time_series_baseline(
     importance = np.array([0] * transformed.shape[1])
     importance[delay_index] = 1
     np.testing.assert_allclose(clf.estimator.feature_importance, importance)
+
+
+@pytest.mark.parametrize("forecast_horizon,gap", [[3, 0], [10, 2], [2, 5]])
+def test_time_series_get_forecast_periods(forecast_horizon, gap, ts_data):
+    X, _, y = ts_data(problem_type=ProblemTypes.TIME_SERIES_REGRESSION)
+    clf = make_timeseries_baseline_pipeline(
+        ProblemTypes.TIME_SERIES_REGRESSION,
+        gap,
+        forecast_horizon,
+        time_index="date",
+    )
+    clf.fit(X, y)
+    result = clf.get_forecast_periods(X)
+
+    assert result.size == forecast_horizon + gap
+    assert result.iloc[0] == X.iloc[-1]["date"] + np.timedelta64(1, "D")
+    assert np.issubdtype(result.dtype, np.datetime64)
+    assert result.name == "date"
+
+
+@pytest.mark.parametrize("forecast_horizon,gap", [[3, 0], [10, 2], [2, 5]])
+def test_time_series_get_forecast_predictions(forecast_horizon, gap, ts_data):
+    X, _, y = ts_data(problem_type=ProblemTypes.TIME_SERIES_REGRESSION)
+
+    X_train, y_train = X.iloc[:15], y.iloc[:15]
+    X_validation = X.iloc[15 : (15 + gap + forecast_horizon)]
```

Comment:
This holdout dataset has a `feature` column that is obviously not generated inside of `get_forecast_predictions()` (i.e. it is multivariate).